tools/plots.py

```diff
@@ -68,17 +68,30 @@ def main():
 
     # Setup plots to produce
     if args.plots == 'all':
-        args.plots = 'tasks clusters cpus stune ediff edspace'
+        args.plots = 'tasks clusters cpus stune ediff edspace peripherals'
 
     # For each rtapp and each run
     if args.outdir is not None:
 
         # Load platform descriptior
         platform = None
-        plt_file = os.path.join(args.outdir, 'platform.json')
```

Comment:
This part should be written properly. I think the old code is also wrong. `plt_file` can never be a directory, so I don't get the whole `isfile` stuff. Seems the below code can be simplified. The `else` path shouldn't be possible

Reply:
I reworked this so that its clearer what's going on. The input is still a directory, but now it loads the traces and platforms a bit more intelligently.